src/benchmark/operators/std_sort_benchmark.cpp

```diff
@@ -0,0 +1,110 @@
+#include <algorithm>
+#include <random>
+
+#include "benchmark/benchmark.h"
+#include "synthetic_table_generator.hpp"
+#include "types.hpp"
+
+#define SIZE 40'000
+
+namespace opossum {
+
+static void BM_StdBaseInt(benchmark::State& state) {
+  std::random_device rd;
+  std::mt19937 mt(rd());
+  std::uniform_int_distribution<int> dist;
+
+  for (auto _ : state) {
+    pmr_vector<int> v;
+    v.reserve(state.range(0));
+    for (int i = 0; i < state.range(0); i++) {
+      v.push_back(dist(mt));
+    }
+  }
+}
+BENCHMARK(BM_StdBaseInt)->Arg(SIZE);
+
+static void BM_StdSortInt(benchmark::State& state) {
+  std::random_device rd;
+  std::mt19937 mt(rd());
+  std::uniform_int_distribution<int> dist;
+
+  for (auto _ : state) {
+    pmr_vector<int> v;
+    v.reserve(state.range(0));
+    for (int i = 0; i < state.range(0); i++) {
+      v.push_back(dist(mt));
+    }
+
+    std::sort(v.begin(), v.end());
+  }
+}
+BENCHMARK(BM_StdSortInt)->Arg(SIZE);
+
+static void BM_StdStableSortInt(benchmark::State& state) {
+  std::random_device rd;
+  std::mt19937 mt(rd());
+  std::uniform_int_distribution<int> dist;
+
+  for (auto _ : state) {
+    pmr_vector<int> v;
+    v.reserve(state.range(0));
+    for (int i = 0; i < state.range(0); i++) {
+      v.push_back(dist(mt));
+    }
+
```

Comment:
You definitely want to look up how to start and stop the performance counters of gbenchmark. This way to benchmark first data creation and then data creation with sorting is extreeeeeemely strange.

Reply:
See our reply to your other comment: We only wanted to get an idea for the performance difference of `std::sort` to `std::stable_sort` here – as opposed to in the sort_benchmark.cpp micro benchmarks, we didn't put much attention here to proper performance counter starting and stopping and instead more or less copied a micro benchmark found in a StackOverflow reply to quickly get some results regarding the comparison of `std::sort` to `std::stable_sort`. But anyway, thanks for your feedback!